### DISCUSSION TITLE
Play Video From Uint8list

### DISCUSSION THREAD

**Discussion by mohammadinfo** [wang-bin/fvp#41] [Ideas]
Hello, could you please add a method to access a content URI? It only supports Android.

**Comment by wang-bin** [comment]
it's already supported, but i haven't tested

**Comment by mohammadinfo** [comment]
I'm waiting for you to publish new version of the package, I opened an issue 😅🙏🏻🌹

**Comment by wang-bin** [comment]
are content uri and uint8list the same thing?

**Comment by mohammadinfo** [comment]
I don't know, but what I really need is a uint8list 🌹.
